ultils/index.js

```diff
@@ -0,0 +1,78 @@
+export function generateID() {
+  return Math.random().toString(36).substring(2);
+}
+
+/**
+ * Get current date, e.g. 2022.10.27 Wednesday
+ * @returns a string to display current date.
+ */
+export function getCurrentDate() {
+  const date = new Date();
+  const year = date.getFullYear();
+  const month = date.getMonth() + 1;
+  const dates = date.getDate();
+  const arr = [
+    "Sunday",
+    "Monday",
+    "Tuesday",
+    "Wednesday",
+    "Thursday",
+    "Friday",
+    "Saturday",
+  ];
+  const day = date.getDay();
+  return year + "." + month + "." + dates + " " + arr[day];
+}
+
+/**
+ *
+ * @returns a string with time format 20:10:36
+ */
+export function displayTime(seconds) {
```

Comment:
please also check replace the 2nd & 3rd if in this function with an if else

Reply:
@Mark-Krensel already changed. Thanks again 👍

Reply:
@Mark-Krensel Hi Mark, i I tried to use if else statement, but something happened when the time was displayed. I think the three if conditions are parallel and can be met at the same time, when man uses else if , the code will go to a condition, once met and will return.